cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt

```diff
@@ -73,17 +73,20 @@ internal interface CampaignManager {
     // Consent Status
     var gdprConsentStatus: GdprCS?
     var ccpaConsentStatus: CcpaCS?
+    var usNatConsentStatus: USNatConsentData?
```

Comment:
I think it is better to rename it to `usNatConsentData`

Reply:
the class with lower case?

Reply:
no, the name of the param itself:
```
var usNatConsentData: USNatConsentData?
```